src/v2i-hub/TimPlugin/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+	"name":"TIM",
+	"description":"Provides Traveller Information Message (TIM).",
+	"version":"3.2.0",
+	"exeLocation":"/bin/TimPlugin",
+	"coreIpAddr":"127.0.0.1",
+	"corePort":24601,
+	"messageTypes":[
+		{
+			"type":"J2735",
+			"subtype":"TIM",
+			"description":"Travelor Information Message indicating the active Work Zone information."
+		}
+	],
+	"configuration":[
+		{
+			"key":"Frequency",
+			"default":"1000",
+			"description":"The frequency to send the TIM in milliseconds."
+		},
+		{
+			"key":"MapFile",
+			"default":"IVP_GF_TIM.xml",
+			"description":""
+		},
+		{
+			"key":"Snap Interval",
+			"default":"300",
+			"description":"The interval in milliseconds to keep a vehicle within a zone before allowing it to transition out of all zones."
+		},
+		{
+			"key":"Vehicle Timeout",
```

Comment:
Same comment as the last one, why does this plugin need to track the vehicle?  If these parameters are not used, remove them.

Reply:
Addressed